Tests/ttLib/tables/data/_c_m_a_p_format_14.ttx

```diff
@@ -3,7 +3,7 @@
 
   <cmap>
     <tableVersion version="0"/>
-    <cmap_format_14 platformID="0" platEncID="5" format="14" length="49" numVarSelectorRecords="2">
+    <cmap_format_14 platformID="0" platEncID="5">
       <map uvs="0xfe00" uv="0x30" name="zero.slash"/>
       <map uvs="0xfe01" uv="0x30" name="None"/>
```

Comment:
I think `name="None"` should go as well, since `None` is a legitimate glyph name, the absence of a `name` should be treated as `None` or something like that.

Reply:
Right, I meant to do that and now forgot, thanks!

Reply:
maybe only upon reading a ttx file, you could check if a glyph named "None" actually exist in the font, and if not, treat that special string as it was, to not break backward compatibility with files produced with older fonttools?

Reply:
Hm, it would be possible if we postpone that test to compile-time. I don't think we know which glyphs are in the font when parsing the xml. Hmhm.

Reply:
hmmmmmm

Reply:
Hm actually, we have a TTFont instance, and I think ttFont.getGlyphOrder() can't fail at this stage. At least not for a full TTX load, as the `<GlyphOrder>` element always comes first.

When merging a partial TTX file with an existing font there is also a valid glyph order. Right now I can't think of a case where this would fail.